### PR TITLE
storybook 7

### PR DESCRIPTION
**Pull request by abernier**
- write your component story into: `.storybook/stories/*.stories.{jsx,mdx}`
- deploys static Storybook to [pmndrs.github.io/react-postprocessing](https://pmndrs.github.io/react-postprocessing) when new release

https://github.com/pmndrs/react-postprocessing/assets/76580/8e8c69a2-8dae-4601-878c-eb20fc55fe7e

### CODE REVIEW

**Comment by talentlessguy** [comment]
storybook is kinda heavy imo, I think it's better to use StackBlitz with git integration

**Comment by abernier** [comment]
> storybook is kinda heavy imo, I think it's better to use StackBlitz with git integration

sb7 is really lighter/faster, on my machine: `yarn storybook` starts in 2s

It is currently configured with `@storybook/react-vite`

**Comment by talentlessguy** [comment]
with vite it's much better cuz less deps and faster

File: .storybook/stories/Autofocus.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import * as THREE from 'three'
+import type { Meta, StoryObj } from '@storybook/react'
+import { Setup } from '../Setup'
+
+import { EffectComposer } from '../../src'
+
+import { Autofocus } from '../../src'
```
Comment (talentlessguy) [inline .storybook/stories/Autofocus.stories.tsx:8]: you can import both in a single statement

File: package.json [hunk 2]
```diff
@@ -67,17 +78,21 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-storybook": "^0.6.12",
     "husky": "^8.0.3",
     "lint-staged": "^13.2.2",
     "prettier": "^2.8.8",
+    "prop-types": "^15.8.1",
```
Comment (talentlessguy) [inline package.json:85]: what do we need prop types for when we already use TS?

**Pull request merged**